Models/rotors/fenestronanimation.xml

```diff
@@ -25,4 +25,43 @@
     <inherits-from>Effects/model-transparent</inherits-from>
     <object-name>rotor_disc_T</object-name>
   </effect>
+
+  <animation>
+    <!-- Rotate the fenestron blades -->
+    <type>rotate</type>
+    <object-name>Tblade</object-name>
+    <property>rotors/tail/blade/position-deg</property>
+    <factor>1</factor>
+    <center>
+      <x-m>0</x-m>
+      <y-m>0</y-m>
+      <z-m>0</z-m>
+    </center>
+    <axis>
+      <x>0</x>
+      <y>1</y>
+      <z>0</z>
+    </axis>
+  </animation>
+  <animation>
+    <!-- Hide the fenestron blades when rotating with high rpm -->
+    <type>select</type>
```

Comment:
I think the separation is not really necessary, but it felt right because the main rotor uses its own model, too.
On the other hand, the doors are animated, too, and they are not separated either.
Let me know if I should skip the separation and animate the fenestron directly in the main model.